src/index.css

```
@@ -6,36 +6,46 @@
-    --input-bg: 10 15 26;
-    --input-border: 55 65 81;
     --input-text: 249 250 251;
     --input-placeholder: 156 163 175;
-    --shadow-card: 0 14px 32px rgb(0 0 0 / 35%);
-    --shadow-glass: 0 20px 44px rgb(0 0 0 / 50%);
+    --shadow-glass: 0 20px 44px rgb(0 0 0 / 40%), 0 0 1px rgb(255 255 255 / 10%);
   }
 
   body {
     @apply min-h-screen m-0;
+    /* Base background color */
```

Stylelint (17.2.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)